br/pkg/lightning/restore/restore.go

```diff
@@ -115,7 +115,8 @@ const (
 		status  VARCHAR(32) NOT NULL,
 		state   TINYINT(1) NOT NULL DEFAULT 0 COMMENT '0: normal, 1: exited before finish',
 		source_bytes BIGINT(20) UNSIGNED NOT NULL DEFAULT 0,
-		cluster_avail BIGINT(20) UNSIGNED NOT NULL DEFAULT 0,
```

Comment:
For existing clutters，this table has already been created, and the table will only be created if the table doesn't exist. This means the table schema will remain the old one without those two new fields for those existing clusters.

Reply:
Lightning is once-only used. It is fine?